src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -27,7 +27,6 @@ const ButtonGroup = createClass({
 	components: {
 		Button: createClass({
 			displayName: 'ButtonGroup.Button',
-			propName: 'Button',
```

Comment:
What is the implication of removing a `propName`? Does that mean the child component shouldn't be pass in as a prop, but only as a child?